app/controllers/request-to-go-live/organisation-address/post.controller.js

```diff
@@ -147,7 +168,7 @@ const submitForm = async function (form, serviceExternalId, correlationId, isReq
         .replace(validPaths.merchantDetails.url, form[clientFieldNames.url])
         .replace(validPaths.merchantDetails.name, form[clientFieldNames.name])
     }
-    return updateService(serviceExternalId, updateRequest.formatPayload(), correlationId)
+    return updateService(req.service.externalId, updateRequest.formatPayload(), req.correlationId)
```

Comment:
Not related to this but for previous PR.

Can this be moved out of `if..else..` as they are same

https://github.com/alphagov/pay-selfservice/pull/3378/files#diff-c974702fa9920cb6df010098f1d670e6fae26f9d5135124c0ca5775d929a56b1L141-L142

https://github.com/alphagov/pay-selfservice/pull/3378/files#diff-c974702fa9920cb6df010098f1d670e6fae26f9d5135124c0ca5775d929a56b1L146-L147

Reply:
Done - thanks